app/src/components/v-input.vue

```diff
@@ -162,6 +184,55 @@ function trimIfEnabled() {
 	}
 }
 
+const MAX_BIG_INT = BigInt(9223372036854775807n);
+const MIN_BIG_INT = BigInt(-9223372036854775807n);
```

Comment:
Are we sure all databases are limited to the int64 sizing when working with BigInt? JS at least is not limited to the `max` and `min` defined here.
```const larger = 9223372036854775807n * 9223372036854775807n;```

Reply:
I pulled this from google. Looks like it's the maximum value for all sql databases.
I even got an error when trying to write a number larger than this from the database side.

Reply:
You can check it out by typing it in the browser itself. After the max number it'll just append some 0s

Reply:
Perfect, i knew this was the case for Postgres hadnt looked beyond that :D

Reply:
Cool 👍 Can we add a small corresponding comment?

Reply:
I think having it as a utility class will be much useful `BigInteger.ts`. 
There's also a project for bigInteger https://github.com/MikeMcl/bignumber.js. 
But I don't think we use much of the features. So What do you say?

Reply:
Also looks like the `system-raw-editor` uses `codemirror` to edit the value. 
I have to take a look into the docs on how to add custom validation there

Reply:
> I think having it as a utility class will be much useful like new BigInteger().
> There's also a project for bigInteger https://github.com/MikeMcl/bignumber.js.

What would this library add over the JS built-in BigInt type? 🤔

Reply:
For our use case not so much.